### PR TITLE
Remove old code

### PR DESCRIPTION
**Pull request by pcGarethJames**
Removed old code related to search and product listing functionality that is no longer supported

### CODE REVIEW

File: includes/admin/pureclarity-admin.php [hunk 1]
```diff
@@ -732,36 +602,6 @@ private function get_advanced_fields() {
...
-			'prodlist_selector_callback',
-			false,
-		);
-		$shopEnabledCheckbox        = array(
-			'pureclarity_shop_enabled',
-			'Enable Shop',
-			'shop_enabled_callback',
-			true,
-		);
-		$shopSelector               = array(
```
Comment (jcbarton) [inline includes/admin/pureclarity-admin.php:759]: what is the shopSelector used for?

File: includes/pureclarity-template.php [hunk 2]
```diff
@@ -55,20 +55,8 @@ public function __construct( &$plugin ) {
 	 * Renders configuration json
 	 */
 	public function render_pureclarity_json() {
-		$style = '';
-		if ( $this->is_pureclarity_active()
```
Comment (jcbarton) [inline includes/pureclarity-template.php:59]: do we not still have if PC is enabled?

File: js/pc.js [hunk 3]
```diff
@@ -12,64 +12,6 @@ var PureClarity = {
...
-
-            var pcContainer = document.createElement('div');
-            var wrapper = document.createElement('div');
-            jQuery(pcContainer).addClass('shop');
-            jQuery(wrapper).addClass('pureclarity-wrapper');
-            jQuery(pcContainer).addClass('pureclarity-container').attr("data-pureclarity", "navigation_category:-1");
-            jQuery(this.config.shop.domSelector).wrap(wrapper).hide();
-            jQuery(".pureclarity-wrapper").append(pcContainer);
-            jQuery(wrapper).addClass('site-main');
-            if (this.config.shop.bmz1) {
```
Comment (jcbarton) [inline js/pc.js:32]: Do we no longer use this.config.shop.bmz1?

File: includes/admin/pureclarity-admin.php (hunk 1 above)
Comment (pcGarethJames) [reply includes/admin/pureclarity-admin.php:759]: It was a type of product listing page, the shopSelector was for a dom element that we could inject our own product listing on top of. It'd stopped being used in a previous version

File: includes/pureclarity-template.php (hunk 2 above)
Comment (pcGarethJames) [reply includes/pureclarity-template.php:59]: This if statement was only for outputting js to replace search results with our own results. It's no longer needed

File: js/pc.js (hunk 3 above)
Comment (pcGarethJames) [reply js/pc.js:32]: yup, it was only used for inserting bmz content with injected search results from PC, search results bmz still work as intended without this

**Pull request merged**